wasm/validation/expressions.py

```diff
@@ -49,27 +49,22 @@ def validate_expression(expression: Tuple[BaseInstruction, ...],
 
         logger.debug('Validating instruction: %s', instruction)
 
-        # SIDE-EFFECT: Instruction validation is inherently
-        # stateful/side-effect causing.  Both `Context.operand_stack` and
```

Comment:
Do I have this right?

This method is still causing side-effects, but we don't care because we made a defensive copy to the `ExpressionContext` first.